### ISSUE TITLE
Analyzer crashes on a null pointer

### ISSUE THREAD

**Issue by abarth** [dart-lang/sdk#25903]
Sorry, don't have a repro yet.

```
Unhandled exception:
AnalysisException: Cannot compute DART_ERRORS for /usr/local/google/home/abarth/.pub-cache/hosted/pub.dartlang.org/mockito-0.10.1/lib/mockito.dart
Caused by Could not find import element for 'null'
Path to root: ImportDirectiveImpl, CompilationUnitImpl
#0      DeclarationResolver._mismatch (package:analyzer/src/generated/resolver.dart:2750)
#1      DeclarationResolver._findImport (package:analyzer/src/generated/resolver.dart:2635)
#2      DeclarationResolver.visitImportDirective (package:analyzer/src/generated/resolver.dart:2359)
#3      ImportDirectiveImpl.accept (package:analyzer/src/dart/ast/ast.dart:5637)
#4      NodeListImpl.accept (package:analyzer/src/dart/ast/ast.dart:7429)
#5      CompilationUnitImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:2375)
#6      RecursiveAstVisitor.visitCompilationUnit (package:analyzer/dart/ast/visitor.dart:646)
#7      CompilationUnitImpl.accept (package:analyzer/src/dart/ast/ast.dart:2369)
#8      DeclarationResolver.resolve (package:analyzer/src/generated/resolver.dart:2087)
#9      BuildCompilationUnitElementTask.internalPerform (package:analyzer/src/task/dart.dart:896)
#10     AnalysisTask._safelyPerform (package:analyzer/task/model.dart:329)
#11     AnalysisTask.perform (package:analyzer/task/model.dart:229)
#12     AnalysisDriver.performWorkItem (package:analyzer/src/task/driver.dart:274)
#13     AnalysisDriver.computeResult (package:analyzer/src/task/driver.dart:109)
#14     AnalysisContextImpl.computeResult (package:analyzer/src/context/context.dart:657)
#15     AnalysisContextImpl.computeErrors (package:analyzer/src/context/context.dart:605)
#16     AnalyzerImpl.prepareErrors.<anonymous closure> (package:analyzer_cli/src/analyzer_impl.dart:133)
#17     _PerformanceTagImpl.makeCurrentWhile (package:analyzer/src/generated/utilities_general.dart:212)
#18     AnalyzerImpl.prepareErrors (package:analyzer_cli/src/analyzer_impl.dart:131)
#19     AnalyzerImpl._analyzeSync (package:analyzer_cli/src/analyzer_impl.dart:167)
#20     AnalyzerImpl.analyzeSync (package:analyzer_cli/src/analyzer_impl.dart:126)
#21     Driver._runAnalyzer (package:analyzer_cli/src/driver.dart:541)
#22     Driver._analyzeAllImpl (package:analyzer_cli/src/driver.dart:201)
#23     Driver._analyzeAll.<anonymous closure> (package:analyzer_cli/src/driver.dart:144)
#24     _PerformanceTagImpl.makeCurrentWhile (package:analyzer/src/generated/utilities_general.dart:212)
#25     Driver._analyzeAll (package:analyzer_cli/src/driver.dart:143)
#26     Driver.start (package:analyzer_cli/src/driver.dart:129)
#27     main (file:///mnt/data/b/build/slave/dart-sdk-linux-dev/build/sdk/pkg/analyzer_cli/bin/analyzer.dart:11)
#28     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:259)
#29     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:148)


#0      AnalysisContextImpl.computeResult (package:analyzer/src/context/context.dart:661)
#1      AnalysisContextImpl.computeErrors (package:analyzer/src/context/context.dart:605)
#2      AnalyzerImpl.prepareErrors.<anonymous closure> (package:analyzer_cli/src/analyzer_impl.dart:133)
#3      _PerformanceTagImpl.makeCurrentWhile (package:analyzer/src/generated/utilities_general.dart:212)
#4      AnalyzerImpl.prepareErrors (package:analyzer_cli/src/analyzer_impl.dart:131)
#5      AnalyzerImpl._analyzeSync (package:analyzer_cli/src/analyzer_impl.dart:167)
#6      AnalyzerImpl.analyzeSync (package:analyzer_cli/src/analyzer_impl.dart:126)
#7      Driver._runAnalyzer (package:analyzer_cli/src/driver.dart:541)
#8      Driver._analyzeAllImpl (package:analyzer_cli/src/driver.dart:201)
#9      Driver._analyzeAll.<anonymous closure> (package:analyzer_cli/src/driver.dart:144)
#10     _PerformanceTagImpl.makeCurrentWhile (package:analyzer/src/generated/utilities_general.dart:212)
#11     Driver._analyzeAll (package:analyzer_cli/src/driver.dart:143)
#12     Driver.start (package:analyzer_cli/src/driver.dart:129)
#13     main (file:///mnt/data/b/build/slave/dart-sdk-linux-dev/build/sdk/pkg/analyzer_cli/bin/analyzer.dart:11)
#14     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:259)
#15     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:148)
```

**Comment by abarth** [comment]
This is with 1.15.0-dev.4.0

**Comment by abarth** [comment]
Might be fixed in 1.15.0-dev.5.0.  Double-checking.

**Comment by abarth** [comment]
Yep, seems to be fixed.
